back/.rubocop_todo.yml

```diff
@@ -0,0 +1,202 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2022-04-27 14:56:37 UTC using RuboCop version 1.28.2.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 257
+Lint/UselessAssignment:
+  Enabled: false
+
+# Offense count: 330
+# Configuration parameters: IgnoredMethods, CountRepeatedAttributes.
+Metrics/AbcSize:
+  Max: 104
+
+# Offense count: 1059
+# Configuration parameters: CountComments, CountAsOne, ExcludedMethods, IgnoredMethods.
+# IgnoredMethods: refine
+Metrics/BlockLength:
+  Max: 837
+
+# Offense count: 16
+# Configuration parameters: CountComments, CountAsOne, ExcludedMethods, IgnoredMethods.
+Metrics/MethodLength:
+  Max: 68
+
+# Offense count: 28
+RSpec/AnyInstance:
+  Enabled: false
+
+# Offense count: 357
+# Configuration parameters: Prefixes.
+# Prefixes: when, with, without
+RSpec/ContextWording:
+  Enabled: false
+
+# Offense count: 10
+# Configuration parameters: IgnoredMetadata.
+RSpec/DescribeClass:
+  Exclude:
+    - 'engines/commercial/admin_api/spec/graphql/folders_spec.rb'
```

Comment:
Do these paths work both on the local machine (where the file paths start from "back" - e.g. in the code editor) as well as inside the docker containers (e.g. when running in CI)?